ingest/expression_files/dense_ingestor.py

```diff
@@ -14,6 +14,7 @@
 
 try:
     from expression_files import GeneExpression
+    from ingest_files import DataArray
```

Comment:
I believe ```manage-study``` doesn't use ingest pipeline to ingest expression matrices due to prior implementation and performance issues. I'm not sure if it will in the future. I've gone ahead and modeled it after the other file types just for consistency. d1b33d4